ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml

```diff
@@ -134,3 +134,137 @@ postsubmits:
           requests:
             cpu: 10m
       serviceAccountName: ci-operator
+  - agent: kubernetes
+    branches:
+    - private-preview-1
+    decorate: true
+    name: branch-ci-openshift-openshift-azure-private-preview-1-e2e-azure-conformance
```

Comment:
These names are too long and probably invalid 

@petr-muller @bbguimaraes will the job that verifies generated config allow me to use a custom name?

Reply:
This part is fixed: `branch-ci-openshift-openshift-azure-private-preview-1-`, which leaves you 8 characters. The only effect of having a job name longer than 63 characters is that it will be truncated when adding (k8s) labels to the resources.